Viper/Viper/CountryList/CountryTableViewController.swift

```diff
@@ -0,0 +1,62 @@
+// CountryTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с выбором стран для отдыха
+final class CountryTableViewController: UITableViewController {
+    // MARK: - Public properties
+
+    var presenter: CountryListPresenterProtocol?
+
+    // MARK: - Life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUI()
+    }
+
+    // MARK: - Private methods
+
+    private func setUI() {
+        tableView.register(CountryTableViewCell.self, forCellReuseIdentifier: "countryTableCell")
+        tableView.dataSource = self
+        tableView.delegate = self
+    }
+}
+
+// MARK: UITableViewDelegate, UITableViewDataSource
+
+extension CountryTableViewController {
```

Comment:
```suggestion
extension CountryTableViewController {
```
а зачем здесь экстеншн? В TableViewController эти методы уже есть.

Reply:
ну мы так раньше делали, можно и в классе написать функции таблицы либо же через расширение